api/api.gradle.kts

```diff
@@ -12,4 +12,11 @@ dependencies {
     api(group = "org.controlsfx", name = "controlsfx", version = "8.40.11")
     api(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "3.1.7-20170808143930-12-gccfeab5")
     implementation(group = "edu.wpi.first.wpiutil", name = "wpiutil-java", version = "2.0.0-20170808143537-16-gf0cc5d9")
+    runtime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "3.1.7-20170808143930-12-gccfeab5", classifier = "all")
+    testRuntime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "3.1.7-20170808143930-12-gccfeab5", classifier = "all")
+    fun testFx(name: String, version: String = "4.0.+") =
+            create(group = "org.testfx", name = name, version = version)
+    testCompile(testFx(name = "testfx-core"))
+    testCompile(testFx(name = "testfx-junit5"))
```

Comment:
Since this `testCompile` dependency is shared now in both `api` and `app` you should put it there so you don't have duplicate version numbers.